docs/user/metal/install_ipi.md

```diff
@@ -0,0 +1,225 @@
+
+* The `image` section will get completely removed.
+
+* The `hardwareProfile` is currently exposed as a way to allow specifying
+  different hardware parameters for deployment.  By default, we will deploy
+  RHCOS to the first disk, but that may not be appropriate for all hardware.
+  The `hardwareProfile` is the field we have available to change that.  This
+  interface is subject to change.  In the meantime, hardware profiles can be
+  found here:
+  https://github.com/metal3-io/baremetal-operator/blob/master/pkg/hardware/profile.go#L48
+
+```yaml
+apiVersion: v1beta4
+baseDomain: test.metalkube.org
+metadata:
+  name: ostest
+compute:
+- name: worker
+  replicas: 1
+controlPlane:
+  name: master
+  replicas: 3
+  platform:
+    baremetal: {}
+platform:
+  baremetal:
+    apiVIP: 192.168.111.5
+    hosts:
+      - name: openshift-master-0
+        role: master
+        bmc:
+          address: ipmi://192.168.111.1:6230
+          username: admin
+          password: password
+        bootMACAddress: 00:11:07:4e:f6:68
+        hardwareProfile: default
+      - name: openshift-master-1
+        role: master
+        bmc:
+          address: ipmi://192.168.111.1:6231
+          username: admin
+          password: password
+        bootMACAddress: 00:11:07:4e:f6:6c
+        hardwareProfile: default
+      - name: openshift-master-2
+        role: master
+        bmc:
+          address: ipmi://192.168.111.1:6232
+          username: admin
+          password: password
+        bootMACAddress: 00:11:07:4e:f6:70
+        hardwareProfile: default
+    - name: openshift-worker-0
+        role: master
+        bmc:
+          address: ipmi://192.168.111.1:6233
+          username: admin
+          password: password
+        bootMACAddress: 00:11:07:4e:f6:71
+        hardwareProfile: default
+    image:
+      source: "http://172.22.0.1/images/rhcos-ootpa-latest.qcow2"
+      checksum: 2b3b1e19e18627d89da400b63430d5bb
+      deployKernel: http://172.22.0.1/images/ironic-python-agent.kernel
+      deployRamdisk: http://172.22.0.1/images/ironic-python-agent.initramfs
+pullSecret: ...
+sshKey: ...
+```
+
+## Work in Progress
+
+Integration of the `baremetal` platform is still a work-in-progress across
+various parts of OpenShift.  This section discusses key items that are not yet
+fully integrated, and their workarounds.
+
+Note that once this work moves into the `openshift/installer` repository, new
+issues will get created or existing issues will be moved to track these gaps
+instead of the leaving the existing issues against the KNI fork of the installer.
+
+### Deployment of the `baremetal-operator`
+
+The `baremetal-operator` provides the server side of the API used by the
+`baremetal` platform `Machine` actuator
+([cluster-api-provider-baremetal](https://github.com/metal3-io/cluster-api-provider-baremetal)).
+This is currently handled by the
+[08_deploy_bmo.sh](https://github.com/openshift-metal3/dev-scripts/blob/master/08_deploy_bmo.sh)
```

Comment:
links to `master` aren't stable

Reply:
True, though this should go away ASAP, once machine-api-operator integration is done